apps/pdc-frontend/src/components/SurveyLink/index.tsx

```diff
@@ -0,0 +1,24 @@
+import type { ComponentProps, ForwardedRef } from 'react';
+import { forwardRef } from 'react';
+import { AdvancedLink } from '@/components';
+import { generateSurveyURL, getSurveyParams } from '@/util';
+
+interface SurveyLinkProps extends ComponentProps<typeof AdvancedLink> {
+  segment: string;
+  // eslint-disable-next-line no-unused-vars
```

Comment:
Je kunt een keer `no-unused-vars` voor alle typescript files uitzetten in `.eslintrc`, die werkt alleen goed voor vanilla JS. TypeScript heeft z'n eigen checks.

Reply:
Dankjewel! Dat is inderdaad zo better.